dotfiles/neovim/snippets/vimwiki.snippets

```diff
@@ -0,0 +1,6 @@
+snippet standup-notes
+	`strftime('# %Y-%m-%d')`
```

Comment:
pretty sure this means I can call arbitrary lua functions from within a snippet 👀